### DISCUSSION TITLE
post/kakaopay-growth-platform-abtest/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#34] [Announcements]
# post/kakaopay-growth-platform-abtest/

카카오페이에서 AB 테스트 서비스를 직접 만들게 된 과정을 공개합니다. 코루틴 액터, ANTLR 등과 같은 재미있는 기술도 소개합니다.

https://tech.kakaopay.com/post/kakaopay-growth-platform-abtest/

**Comment by janghaeun** [comment]
안녕하세요. 작성해주신 글 잘 읽었습니다. 내부 구현 방식과 AB 테스트 시 요구사항을 잘 살펴볼 수 있었습니다.
한 가지 궁금한 것은 카카오페이만의 AB 테스트를 진행해서 위너 선정까지 마무리 하셨는데 이후 정량적으로 도출된 인사이트가 있을까요?
어떤 화면을 테스트하셨고 어떤 인사이트를 도출하셨는지 궁금합니다:)
